Tasks/XamariniOS/task.json

```diff
@@ -132,6 +138,7 @@
         },
         {
             "name": "iosSigningIdentity",
+            "aliases": [ "signingIosIdentity" ],
```

Comment:
The variable name reads better than the alias here, better one will be "signingIdentity"